### ISSUE TITLE
Some Suggestions

### ISSUE THREAD

**Issue by jackkoenig** [Tyriar/wiki-sync-example-wiki#1]
I loved your guide at http://www.growingwiththeweb.com/2016/07/enabling-pull-requests-on-github-wikis.html

I have a couple of ideas:
1. [It appears that Travis does now support cron jobs](https://docs.travis-ci.com/user/cron-jobs/) on a daily, weekly, or monthly basis. If this is frequent enough (I wish it could run more frequently but oh well), then there is no need for additional tools.
1. [In my inspired implementation](https://github.com/jackkoenig-bot/wiki-sync-example-wiki), since PRs from forks don't have access to secure variables (eg. GITHUB_API_KEY), I have it check if the PR can cleanly merge. I believe your repo will always fail on PRs from forked repos.
1. I think it is a little cleaner to only do the merge when Travis runs on a push to master, but I don't think this has any correctness implications.

Anyway, thank you so much for your guide. It will have to do until Github adds better support for external users contributing to wikis.

**Comment by Tyriar** [comment]
Hi @jackkoenig, glad you liked it 😄 

I saw this in my reading list recently, definitely makes the situation better. I believe they're still in beta, I may fiddle with it and update the article when it's officially launched.

It may fail CI associated with it but GH itself should say whether the merge is clean? Do you mean that you won't know if there are conflicts between the repo and the `-wiki` repo? I really like the improvements you made to the [.travis.yml](https://github.com/jackkoenig-bot/wiki-sync-example-wiki/blob/master/.travis.yml), I'll create an (internal) issue to track adding that to the article 😄

**Comment by jackkoenig** [comment]
Makes sense to wait until official launch. I'm trying it out now on my example; hopefully it works well!

You're right that while Travis fails, GitHub does tell you that there are no conflicts so it can be merged. I did an example PR on this repo here:  https://github.com/Tyriar/wiki-sync-example-wiki/pull/2. In my repo, if you do such a PR, Travis will pass as well because it will just check that the merge is clean. It's obviously not a huge deal, I just think it's a little cleaner on the commit history to not have X's from Travis 😄

**Comment by Tyriar** [comment]
Yeah it's really cool checking for a clean merge on the PR, reduces the likelyhood of a conflict significantly.

**Comment by bsanders** [comment]
@Tyriar @jackkoenig Thank you both for putting this together!
